AniHyou/Screens/StaffDetails/Views/StaffDetailsView.swift

```diff
@@ -110,7 +110,15 @@ struct StaffDetailsView: View {
     @ViewBuilder
     var staffMedia: some View {
         LazyVStack(alignment: .leading) {
-            Toggle("On my list", isOn: $viewModel.mediaOnMyList)
+            HStack {
```

Comment:
no need to create a HStack with an extra Text. Use just the picker.

Reply:
I'll try using that picker you created.. the `Picker` element wasn't putting the label on the screen for some reason which is why I used that stack